examples/.experimental/next-app-dir/src/server/trpc.ts

```diff
@@ -25,7 +25,8 @@ const t = initTRPC.context<Context>().create({
 export const router = t.router;
 export const publicProcedure = t.procedure;
 
-export const createAction = experimental_createServerActionHandler(t, {
+export const createAction = experimental_createServerActionHandler({
+  t,
```

Comment:
signature 1 - for those just using procedure builders as is without using routers or a client - pass in the root t object